hack/local-up-cluster.sh

```diff
@@ -81,6 +81,7 @@ EXTERNAL_CLOUD_PROVIDER=${EXTERNAL_CLOUD_PROVIDER:-false}
 EXTERNAL_CLOUD_PROVIDER_BINARY=${EXTERNAL_CLOUD_PROVIDER_BINARY:-""}
 EXTERNAL_CLOUD_VOLUME_PLUGIN=${EXTERNAL_CLOUD_VOLUME_PLUGIN:-""}
 CONFIGURE_CLOUD_ROUTES=${CONFIGURE_CLOUD_ROUTES:-true}
+CLOUD_CTLRMGR_FLAGS=${CLOUD_CTLRMGR_FLAGS:-""}
```

Comment:
Better use single quotes for static strings

Reply:
To avoid variable expansion in the string?  The other variables all use double quotes, is it a concern for them too?

Reply:
Yes, and yes. :)

Reply:
Makes sense, but how much does it matter for a script meant to be a dev/test tool, where the env var value is usually set by a human running the script in a terminal... in fact, maybe they are using env vars for some values here and want expansion to take place.  All I'm saying is--is it worth it to make the change to other environment variables in this script (if it is, its probably better to do that in a separate, targeted PR anyway).

Reply:
That, is a decision for the maintainer to do.

Reply:
(I see no reasons though, when adding _new_ code, not to do it as well as possible just because there might be legacy code that could do with some touch-up.)